cfy_manager/main.py

```diff
@@ -602,27 +603,28 @@ def _finish_configuration(only_install):
     _print_time()
 
 
+def _get_config_path_from_args():
```

Comment:
I think everywhere that uses the config path arg has config_file passed as an argument doesn't it?